src/encoded/batch_download.py

```diff
@@ -395,6 +414,11 @@ def metadata_tsv(context, request):
         for p in param_list.get('@id', [])
     ]
     qs.drop('limit')
+
+    # Determine if "visualizable=true" in query string to select only visualizable files.
+    visualizable_only = 'true' in param_list.get('visualizable', [])
```

Comment:
I think you can also do:

```
visualizable_only = qs.is_param('visualizable', 'true')
```

https://github.com/ENCODE-DCC/snovault/blob/02f7a9cbb70fa21964fbc445ab41d88fa91580a5/src/snovault/elasticsearch/searches/parsers.py#L31

Reply:
Oh wow. `qs.is_param` makes that way cleaner. Thanks!